.github/workflows/cacerts-apply-patch.sh

```diff
@@ -6,13 +6,19 @@ source "${GHA_SCRIPTS_DIR}/.github/workflows/common.sh"
 
 prepare_git_repo
 
+if ! check_remote_branch "cacerts-${VERSION_NEW}-${TARGET_BRANCH}"; then
+    echo 'UPDATE_NEEDED=0' >>"${GITHUB_OUTPUT}"
```

Comment:
I'd say that setting UPDATE_NEEDED to 0 is not necessary, since we always check if UPDATE_NEEDED is 1.